hooks/new

```diff
@@ -52,7 +52,8 @@ ask_for_database() {
       'Where would you like to house Cloud Foundry configuration and metadata?' \
       -o '[postgres-db]  PostgreSQL Amazon RDS' \
       -o '[mysql-db]     MySQL Amazon RDS' \
-      -o '[local-db]     an internal database node (not HA)'
+      -o '[local-db]     an internal database node (not HA)' \
+      -o '[local-ha-db]     internal HA database cluster'
```

Comment:
Fix the spacing so the labels all line up.

Also, change the name to include that it is a pair of HA database nodes.